db/migrate/20180624192035_add_deleted_at_to_bike_organizations.rb

```diff
@@ -1,4 +1,4 @@
-class AddDeletedAtToBikeOrganizations < ActiveRecord::Migration
+class AddDeletedAtToBikeOrganizations < ActiveRecord::Migration[4.2]
```

Comment:
Rails 5 requires the migration version number to be appended to the migration class name